### PR TITLE
Redesign dapp connector 2

### PR DESCRIPTION
**Pull request by paulclindo**
Overview Tab

UTXO Addresses Tab

## Temporal large screen layout
Screen Shot 2022-03-04 at 12 15 12 PM

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201907324853865
  - https://app.asana.com/0/0/1201907324853872

### CODE REVIEW

File: packages/yoroi-extension/app/ergo-connector/components/signin/CardanoSignTxPage.js [hunk 1]
```diff
@@ -310,145 +280,180 @@ class SignTxPage extends Component<Props, State> {
...
+                    alignItems: 'center',
+                    backgroundColor: '#f8f8f8',
+                    img: {
+                      width: '20px',
+                    },
+                  }}
+                >
+                  <img src={faviconUrl} alt={`${url} favicon`} />
+                </Box>
+              ) : null}
```
Comment (neuodev) [inline packages/yoroi-extension/app/ergo-connector/components/signin/CardanoSignTxPage.js]: We should display a placeholder image.We already have this in the connected websites screen in Yoroi 
[check this ](https://github.com/Emurgo/yoroi-frontend/blob/develop/packages/yoroi-extension/app/components/dapp-connector/ConnectedWebsites/WalletRow.js#L140)

File: packages/yoroi-extension/app/ergo-connector/components/signin/SignTxPage.js [hunk 2]
```diff
@@ -235,141 +239,188 @@ class SignTxPage extends Component<Props, State> {
...
+                    alignItems: 'center',
+                    backgroundColor: '#f8f8f8',
+                    img: {
+                      width: '20px',
+                    },
+                  }}
+                >
+                  <img src={faviconUrl} alt={`${url} favicon`} />
+                </Box>
+              ) : null}
```
Comment (neuodev) [inline packages/yoroi-extension/app/ergo-connector/components/signin/SignTxPage.js]: Should display the placeholder image here as well

File: packages/yoroi-extension/app/ergo-connector/components/signin/SignTxTabs.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+  const tabs = [
+    {
+      id: 0,
+      label: intl.formatMessage(globalMessages.overview),
+      component: overviewContent,
+    },
+    {
+      id: 1,
+      label: 'UTXO addresses',
```
Comment (neuodev) [inline packages/yoroi-extension/app/ergo-connector/components/signin/SignTxTabs.js]: This should be added to `intl` as well.

**Review by neuodev — CHANGES_REQUESTED** [review]

**Comment by zuzunker** [comment]
@AhmedIbrahim336 , please add the fixes for the comments you have left

**Review by neuodev — APPROVED** [review]
LGTM

**Comment by neuodev** [comment]
@Nebyt  review it carefully as Pual deleted some of our style files :)

**Review by Nebyt — CHANGES_REQUESTED** [review]
Several issues should be fixed:
- [x] [The page is shifted up](https://app.asana.com/0/1201126389914030/1201907324853858)
- [x] [Scroll is missed on the signing page](https://app.asana.com/0/1201126389914030/1201907324853865)
- [x] [Fix dApp connector layout on large screens, again](https://app.asana.com/0/1201126389914030/1201907324853872)

**Review by Nebyt — APPROVED** [review]
Checked. :+1:

**Pull request merged**